bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/thing/sink.xml

```diff
@@ -21,6 +21,18 @@
 				<label>Name</label>
 				<description>The name of one specific device.</description>
 			</parameter>
+			<parameter name="activateSimpleProtocolSink" type="boolean">
+				<label>Sink by simple-protocol-tcp</label>
+				<description>Activation of a corresponding sink in openHAB (needs module-simple-protocol-tcp loaded on the server)</description>
+				<default>false</default>
+				<required>false</required>
```

Comment:
The `required` tag is deprecated. You could use the attribute instead: `required="true"`